Assets/Mirror/Runtime/Transport/Websocket/WebsocketTransport.cs

```diff
@@ -11,6 +11,10 @@ public class WebsocketTransport : Transport
 
         public int port;
 
+        public bool Secure = false;
+        public string CertificatePath = "";
```

Comment:
don't pass Path and password,   the user may provide the certificate in some other way.   Instead receive the X509Certificate2,  and it is up to the user how to get it